### PR TITLE
Pastebin converter for multiple links

### PR DESCRIPTION
**Pull request by MLGPenguin**
Implemented support for multiple links closing #189 and fulfilling my destiny. I've hard-capped the amount of auto-converted links in one message to 6 as anything more than that seems unnecessary.

.env was modified.

### CODE REVIEW

**Comment by stephendotgg** [comment]
Nice

File: src/main/kotlin/com/learnspigot/bot/Server.kt [hunk 1]
```diff
@@ -19,5 +19,6 @@ object Server {
     val upvoteEmoji = Emoji.fromCustom("upvote", Environment.get("UPVOTE_EMOJI_ID").toLong(), false)
     val downvoteEmoji = Emoji.fromCustom("downvote", Environment.get("DOWNVOTE_EMOJI_ID").toLong(), false)
     val nostarboardEmoji = Emoji.fromCustom("nostarboard", Environment.get("NOSTARBOARD_EMOJI_ID").toLong(), false)
+    val rightarrowEmoji = Emoji.fromCustom("right", Environment.get("RIGHT_ARROW_EMOJI_ID").toLong(), false)
```
Comment (joshdevous) [inline src/main/kotlin/com/learnspigot/bot/Server.kt]: rightArrow 🥺

**Review by joshdevous — APPROVED** [review]
All looks good bro just the lowercase a making my stomach squirm. I hate "nostarboard" too but it is kind of the name...

File: src/main/kotlin/com/learnspigot/bot/Server.kt (hunk 1 above)
Comment (MLGPenguin) [reply src/main/kotlin/com/learnspigot/bot/Server.kt]: I mean I was just copying the others, trust me, this wasn't what I wanted either

**Review by joshdevous — APPROVED** [review]

**Pull request merged**